src/Context/Attachments.ts

```diff
@@ -1,20 +1,81 @@
-import { TestContext } from './TestContext';
+import * as assert from 'assert';
 import * as fs from 'fs';
 import * as path from 'path';
+import { Md5 } from '../Utils/MD5';
+import { TestContext } from './TestContext';
 
 const testResultsEnvVar = 'JSTEST_RESULTS_DIRECTORY';

```

Comment:
Can you expose this so that we can access this in jstestadapter when settings attachments folder.

Reply:
Will do in the coming PRs